src/elements/breadcrumbs/src/index.tsx

```diff
@@ -2,24 +2,31 @@
 
 import * as React from 'react'
 import { jsx, Styled, useThemeUI } from 'theme-ui'
+import get from 'lodash/get'
 import { Icon } from '@uswitch/trustyle.icon'
 
-const HomeIcon: React.FC = () => {
+const lookup = (variant: string) =>
+  variant === 'base' ? 'breadcrumbs2.base' : `breadcrumbs2.variants.${variant}`
+
+interface HomeIconProps extends React.HTMLAttributes<HTMLDivElement> {
+  variant?: 'base' | 'light'
+}
+
+const HomeIcon: React.FC<HomeIconProps> = ({ variant = 'base' }) => {
   const { theme }: any = useThemeUI()
 
+  const iconColor = get(theme, `${lookup(variant)}.homeIcon.color`)
```

Comment:
Thoughts on using `eval` or `new Function` for this? Seems pointless to load a library for what is a completely safe use of eval.

Reply:
write a utiltiy helper to use eval behind scenes so people knwo why we use it